utils/ami-cleanup/ami_cleanup.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
```

Comment:
this file is failing on some PEP8 linting checks. Could you use [black](https://github.com/psf/black) or your IDE please?